src/main/java/com/mraof/minestuck/data/recipe/MinestuckCombinationsProvider.java

```diff
@@ -12,6 +12,7 @@
 import net.minecraft.world.item.DyeColor;
 import net.minecraft.world.item.Item;
 import net.minecraft.world.item.Items;
+import net.minecraft.world.item.crafting.Ingredient;
```

Comment:
Remove unused import

Reply:
im.. n0t sure where that came fr0m. i d0nt recall adding it myself. either way, will rem0ve!